README.md

```diff
@@ -70,7 +70,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://www.loom.com/share/b238b8c6c96a447ea9de1bdbe7c3da47)
+- [Live Demo](https://budget-app-production-969d.up.railway.app/)
```

Comment:
- [ ] when trying to open de live demo the user gets a server error, please make sure to fix it before requesting another review.